hut_10sqft/config/emacs/emacs_130s-brya.el

```diff
@@ -0,0 +1,10 @@
+; .emacs specific for 130s-brya ChromeOS
+; 20231123 Originally copied from emacs_130s-p16s.el
+
+(load "~/.config/hut_10sqft/hut_10sqft/config/emacs/emacs.el")
```

Comment:
This will probably break Emacs on ChromeOS. Can be addressed in https://github.com/kinu-garage/hut_10sqft/pull/961